presidio-anonymizer/tests/test_anonymizer_engine.py

```diff
@@ -17,7 +17,7 @@
 
 def test_given_request_anonymizers_return_list():
     engine = AnonymizerEngine()
-    expected_list = ["hash", "mask", "redact", "replace", "encrypt"]
+    expected_list = ["hash", "mask", "redact", "replace", "encrypt" "replace", "custom"]
```

Comment:
Replace appear twice and a comma is missing...
```suggestion
    expected_list = ["hash", "mask", "redact", "replace", "encrypt", "custom"]
```

Reply:
fixed